plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -1500,17 +1545,25 @@ def update_auth_policy_server(self, authentication_policy_server):
 
             # Authentication and Policy Server exists, check update is required
             # Edit API not working, remove this
-            self.format_payload_for_update(self.have.get("authenticationPolicyServer")[auth_server_index].get("details"),
-                                           self.want.get("authenticationPolicyServer")[auth_server_index]).check_return_status()
-            is_ise_server_enabled = self.have.get("authenticationPolicyServer")[auth_server_index].get("details").get("isIseEnabled")
-            if not (is_ise_server_enabled or self.requires_update(self.have.get("authenticationPolicyServer")[auth_server_index].get("details"),
-                                                                  self.want.get("authenticationPolicyServer")[auth_server_index],
-                                                                  self.authentication_policy_server_obj_params)):
+            have_auth_server_details = self.have.get("authenticationPolicyServer")[auth_server_index].get("details")
+            want_auth_server_details = self.want.get("authenticationPolicyServer")[auth_server_index]
+
```

Comment:
self.log("Formatting payload for update between current and desired authentication server details.", "DEBUG")